src/napari_micromanager/_gui_objects/_toolbar.py

```diff
@@ -394,3 +191,151 @@ def _add_dock_widget(
+        self.addSubWidget(QLabel(text="Exposure:"))
+        self.addSubWidget(DefaultCameraExposureWidget())
+
+
+class SnapLiveToolBar(MMToolBar):
+    def __init__(self, parent: QWidget) -> None:
+        super().__init__("Snap Live", parent)
+        snap_btn = SnapButton()
+        snap_btn.setText("")
+        snap_btn.setToolTip("Snap")
+        snap_btn.setFixedSize(TOOL_SIZE, TOOL_SIZE)
+        self.addSubWidget(snap_btn)
+
+        live_btn = LiveButton()
+        live_btn.setText("")
+        live_btn.setToolTip("Live Mode")
+        live_btn.button_text_off = ""
+        live_btn.button_text_on = ""
+        live_btn.setFixedSize(TOOL_SIZE, TOOL_SIZE)
+        self.addSubWidget(live_btn)
+
+
+class ToolsToolBar(MMToolBar):
+    """A QToolBar containing QPushButtons for pymmcore-widgets.
+
+    e.g. Property Browser, GroupPresetTableWidget, ...
+
+    QPushButtons are connected to the `_show_dock_widget` method.
+
+    The QPushButton.whatsThis() property is used to store the key that
+    will be used by the `_show_dock_widget` method.
+    """
+
+    def __init__(self, parent: MicroManagerToolbar) -> None:
+        super().__init__("Tools", parent)
+        self.groupbox.setStyleSheet("border: 0px;")
+
+        if not isinstance(parent, MicroManagerToolbar):
+            raise TypeError("parent must be a MicroManagerToolbar instance.")
+
+        for key in DOCK_WIDGETS:
+            btn_icon = DOCK_WIDGETS[key][1]
+            if btn_icon is None:
+                continue
+
+            btn = QPushButton()
+            btn.setToolTip(key)
+            btn.setFixedSize(TOOL_SIZE, TOOL_SIZE)
+            btn.setIcon(icon(btn_icon, color=(0, 255, 0)))
+            btn.setIconSize(QSize(30, 30))
+            btn.setWhatsThis(key)
+            btn.clicked.connect(parent._show_dock_widget)
+            self.addSubWidget(btn)
+
+
+class ShuttersToolBar(MMToolBar):
+    def __init__(self, parent: QWidget) -> None:
+        super().__init__("Shutters", parent)
+        self.addSubWidget(MMShuttersWidget())
+        self.groupbox.setStyleSheet("border: 0px;")
+
+
+class PluginsToolBar(MMToolBar):
+    """A QToolBar containing plugins QPushButtons.
+
+    e.g. MDA, ...
+
+    QPushButtons are connected to the `_show_dock_widget` method.
+
+    The QPushButton.whatsThis() property is used to store the key that
+    will be used by the `_show_dock_widget` method.
+    """
+
+    def __init__(self, parent: MicroManagerToolbar) -> None:
+        super().__init__("Plugins", parent)
+        self.addSubWidget(MMShuttersWidget())
+        self.groupbox.setStyleSheet("border: 0px;")
+
+        if not isinstance(parent, MicroManagerToolbar):
+            raise TypeError("parent must be a MicroManagerToolbar instance.")
+
+        btn = QPushButton("MDA")
+        btn.setToolTip("MultiDimensional Acquisition")
+        btn.setMinimumHeight(TOOL_SIZE)
+        btn.setWhatsThis("MDA")
+        btn.clicked.connect(parent._show_dock_widget)
```

Comment:
adding the button to the toolbar
```suggestion
      btn.clicked.connect(parent._show_dock_widget)
      self.addSubWidget(btn)
```